bitnami/prometheus-operator/templates/prometheus/prometheus.yaml

```diff
@@ -137,6 +137,11 @@ spec:
     {{- if .Values.prometheus.podAffinity }}
     podAffinity: {{- include "prometheus-operator.tplValue" (dict "value" .Values.prometheus.podAffinity "context" $) | nindent 6 }}
     {{- end }}
+    {{- if or .Values.prometheus.additionalScrapeConfigsExternal }}
```

Comment:
Please document this paramater in the readme

Reply:
The existence of the flag is documented in the readme, but it wasn't used before. Should I document the semantics of this change (conventional name of the ConfigMap) instead?

Reply:
Yea. The ConfigMap name and key should be mentioned here.